src/t2iapi/combined/combined_requests.proto

```diff
@@ -29,3 +31,30 @@ message CreateContextStateWithAssociationAndSetOperatingModeRequest {
                                             // shall be set
     t2iapi.operation.OperatingMode operating_mode = 4;  // the requested @OperatingMode value of the operation state
 }
+
+/*
+Request to set the ActivationState of the component or metric with the given handle to the requested
+ComponentActivation value and to set the OperatingMode of the operation with the given handle to the requested
+OperatingMode value.
+*/
+message SetComponentActivationAndSetOperatingModeRequest {
+    string component_metric_descriptor_handle = 1; // descriptor handle of the descriptor for which the ActivationState
+                                                   // shall be set in the corresponding state element
+    t2iapi.activation_state.ComponentActivation activation= 2; // value to set as the ActivationState value
+    string operation_descriptor_handle = 3; // the operation descriptor handle for which the @OperatingMode value
+    // shall be set
+    t2iapi.operation.OperatingMode operating_mode = 4; // the requested @OperatingMode value of the operation state
+}
+
+/*
+Request to set the ActivationState of the alert with the given handle to the requested AlertActivation value and
+to set the OperatingMode of the operation with the given handle to the requested OperatingMode value.
+*/
+message SetAlertActivationAndSetOperatingModeRequest {
+    string alert_descriptor_handle = 1; // descriptor handle of the descriptor for which the ActivationState shall be
+                                        // set in the corresponding state element
+    t2iapi.activation_state.AlertActivation activation= 2; // value to set as the ActivationState value
+    string operation_descriptor_handle = 3; // the operation descriptor handle for which the @OperatingMode value
+                                            // shall be set
+    t2iapi.operation.OperatingMode operating_mode = 4; // the requested @OperatingMode value of the operation state
+}
```

Comment:
in both new mesages - adapt the wordning of the attributes to be consistent, see also CreateContextStateWithAssociationAndSetOperatingModeRequest